### ISSUE TITLE
Teammate for Presentation and Tutorial

### ISSUE THREAD

**Issue by georgewbar** [KTH/devops-course#417]
I need a teammate for presentation for next week (week 3, a time slot is already booked) and for the tutorial because my teammate will not continue the course. I have taken Software Engineering fundementals and Software reliability courses (and of course the mandatory courses in master in Computer science program). These two courses are the courses that are mostly related to DevOps and automated testing.

I am flexible with the presentation topic but I would prefer if someone would like to learn more about the basics and have a presentation about comparisons or something like this. However, i am also open to suggestions. Feel free to comment here, and i will get a notification!

Same thing regarding the tutorial topic. I am flexible regarding the topics fot this as well.

**Comment by dieflo4711** [comment]
Hi! I'm also looking for a teammate for the presentation (just the presentation) next week. I have also taken the Software Engineering fundamentals course.
Contact me if you're interested:
Mail: dleon@kth.se
Facebook: /diego.floresleon

**Comment by georgewbar** [comment]
I have found a partner for the presentation.
I need a teammate for the tutorial.

**Comment by sujon2100** [comment]
> I have found a partner for the presentation.
> I need a teammate for the tutorial.

Hi, 

I am looking for a teammate too. Can we make a team for the presentation? I need also a teammate for the tutorial and demo too. Let me know what is your situation. Thanks!
Mail: mhud@kth.se

**Comment by georgewbar** [comment]
Regarding the presentation, I contacted my partner in presentation. We came to the conclusion that it will be harder to coordinate between 3 persons when it comes to the presentation (as you probably have see today when one person has to coordinate the slides and the time constraints become harder to stick to).

Regarding the tutorial, I need a teammate, and i would be happy to cooperate. I will send an email to you now as well with my facebook.
